src/Sylius/Bundle/CoreBundle/EventListener/Workflow/Order/CancelOrderListener.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\CoreBundle\EventListener\Workflow\Order;
+
+use Sylius\Component\Core\Inventory\Operator\OrderInventoryOperatorInterface;
+use Sylius\Component\Core\Model\OrderInterface;
+use Symfony\Component\Workflow\Event\CompletedEvent;
+use Webmozart\Assert\Assert;
+
+final class CancelOrderListener
```

Comment:
```suggestion
final class RollbackInventoryListener
```
  It is only suggestion, but `CancelOrderListener` is wrong for me, because there is `HoldInventoryListener`

Reply:
yeah i would go maybe for some GiveBackOrderInventoryListener 🤔